baitap-submit/toidicodedao/01-prompt-engineering/01-prompt-engineering.md

```diff
@@ -1,11 +1,56 @@
 - Prompt 1
-
+  As a teching assistance, make a list of multiple choice question has at least 10 questions about a lesson. The lesson has content delimited by quote:
+  "
+  [Lesson Content]
+  "
 - Prompt 2
-
+  I have a short paragraph, analyze the meaning of the paragraph in the double quotes
```

Comment:
Mình nên có set role cho model, việc đó sẽ giúp set tone và mức độ expert của model cho chủ đề này, ví dụ expert in literature writer. đoạn văn nên set thành template như {{text}}.